src/wp-includes/class-wp-scripts.php

```diff
@@ -390,8 +390,12 @@ public function do_item( $handle, $group = false ) {
 			return true;
 		}
 
+		$strategy = $this->get_eligible_loading_strategy( $handle );
+		if( '' !== $strategy ) {
+			$strategy = ' '.$strategy;
```

Comment:
```suggestion
			$strategy =  ' ' . $strategy;
```

I'd usually prefer to see a `sprintf` approach here, but I'm also conscious of the fact that we are never injecting anything here other than a space, so we can probably get by with this. Not going to fail the review based on this alone.